e2e/cypress/integration/system_console/settings_spec.js

```diff
@@ -0,0 +1,241 @@
+                cy.apiLDAPTest();
+                cy.apiLDAPSync();
+
+                return cy.apiGetConfig();
+            }).then(({config}) => {
+                testSettings = setLDAPTestSettings(config);
+                testSettings.user = ldapUser;
+                return cy.doLDAPLogin(testSettings);
+            }).then(() => {
+                return cy.apiLogout();
+            }).then(() => {
+                return cy.apiAdminLogin();
+            }).then(() => {
+                cy.visit('/admin_console');
+                return cy.apiCreateUser();
+            }).then(({user: user2}) => {
+                // # Create SAML user
+                samlUser = user2;
+                const body = {
+                    from: 'email',
+                    auto: false,
+                };
+                body.matches = {};
+                body.matches[user2.email] = user2.username;
+
+                return migrateAuthToSAML(body);
+            }).then(() => {
+                return cy.apiGenerateMfaSecret(mfaUser.id);
+            }).then((res) => {
+                // # Create MFA user
+                const token = authenticator.generateToken(res.code.secret);
+
+                return cy.apiActivateUserMFA(mfaUser.id, true, token);
+            });
+    });
+
+    it('MM-T953: Verify correct authentication method', () => {
+        cy.visit('/admin_console/user_management/users');
+
+        // # Type sysadmin
+        cy.get('#searchUsers').clear().type('sysadmin').wait(TIMEOUTS.HALF_SEC);
+
+        // * Verify sign-in method
+        cy.findByTestId('userListRow').within(() => {
+            cy.get('.more-modal__details').
+                should('be.visible').
+                and('contain.text', 'Sign-in Method: Email');
+        });
+
+        // # Type saml user
+        cy.get('#searchUsers').clear().type(samlUser.username).wait(TIMEOUTS.HALF_SEC);
+
+        // * Verify sign-in method
+        cy.findByTestId('userListRow').within(() => {
+            cy.get('.more-modal__details').
+                should('be.visible').
+                and('contain.text', 'Sign-in Method: SAML');
+        });
+
+        // # Type ldap user
+        cy.get('#searchUsers').clear().type(ldapUser.username).wait(TIMEOUTS.HALF_SEC);
+
+        // * Verify sign-in method
+        cy.findByTestId('userListRow').within(() => {
+            cy.get('.more-modal__details').
+                should('be.visible').
+                and('contain.text', 'Sign-in Method: LDAP');
+        });
+
+        // # Type mfa user
+        cy.get('#searchUsers').clear().type(mfaUser.username).wait(TIMEOUTS.HALF_SEC);
+
+        // * Verify sign-in method
+        cy.findByTestId('userListRow').within(() => {
+            cy.get('.more-modal__details').
+                should('be.visible').
+                and('contain.text', 'MFA: Yes');
+        });
+    });
+
+    it('MM-T1149: Hide mobile-specific settings', () => {
```

Comment:
Please move this into a separate spec file with test metadata of:
```
// Group: @system_console @te_only
```

We'll run this against a test server without license or team edition. Therefore, no need to add step to remove the license, though it's mentioned in the test case.